### ISSUE TITLE
dangerouslyAllowMutability binding

### ISSUE THREAD

**Issue by happylinks** [bloodyowl/rescript-recoil#24]
Hey!

I needed the dangerouslyAllowMutability binding for something, any interest to have a PR for it?

```
type atomWithDangerouslyAllowMutabilityConfig('value) = {
  key: string,
  default: 'value,
  dangerouslyAllowMutability: bool,
};

[@bs.module "recoil"]
external atomWithDangerouslyAllowMutability:
  atomWithDangerouslyAllowMutabilityConfig('value) =>
  Recoil.readWrite('value) =
  "atom";
```

Not sure if this is the nicest API for it, and I don't think a lot of people will need it, but thought I'd ask :)

**Comment by bloodyowl** [comment]
Sure! I'd happily merge a PR for that 😀

**Comment by mcharnet** [comment]
Hey ! 

First of all, thanks for the library which let me discover and use Recoil everyday in my Rescript project !

I also need this binding. Any update on this @happylinks, @bloodyowl ?

Thanks a lot !

**Comment by happylinks** [comment]
Took a little while, but I just opened a PR for this: https://github.com/bloodyowl/rescript-recoil/pull/29

**Comment by bloodyowl** [comment]
Thanks a lot @happylinks! Your feature has been released in `v2.1.0`
